src/DataFixtures/PrivatePackageDownloadFixtures.php

```diff
@@ -52,6 +52,7 @@ private function loadDownloads(PackageName $package): void
         $progress = new ProgressBar($output, 30);
         $output->writeln(sprintf('Package: %s', $package->name()));
         $progress->start();
+        $versions = array_map(fn () => $this->faker->numerify('#.#.#'), range(1, 10));
```

Comment:
there will be less granulation, otherwise it is after 500 versions which is rare ... or maybe not :thinking: